src/azul/__init__.py

```diff
@@ -148,6 +148,10 @@ def share_es_domain(self) -> bool:
     def s3_bucket(self) -> str:
         return self.environ['AZUL_S3_BUCKET']
 
+    @property
```

Comment:
The name also shouldn't imply what the return value is. Otherwise we would need to change the name whenever we change the value. The point of an abstraction is that you can change its implementation without affecting code that uses the abstraction.